django/db/models/options.py

```diff
@@ -250,7 +251,11 @@ def _prepare(self, model):
                 field.primary_key = True
                 self.setup_pk(field)
             else:
-                auto = AutoField(verbose_name='ID', primary_key=True, auto_created=True)
+                pk_setting = getattr(self.app_config, "default_auto_field", settings.DEFAULT_MODEL_PK)
+                pk_class = import_string(pk_setting)
+                if not issubclass(pk_class, AutoField):
+                    raise ValueError(f"Primary key {pk_class} is not a subclass of AutoField")
+                auto = pk_class(verbose_name='ID', primary_key=True, auto_created=True)
```

Comment:
```suggestion
                pk_setting = getattr(self.app_config, 'default_auto_field', settings.DEFAULT_AUTO_FIELD)
                pk_class = import_string(pk_setting)
                if not issubclass(pk_class, AutoField):
                    raise ValueError("Configured default auto field '%s' is not a subclass of AutoField." % pk_class)
                auto = pk_class(verbose_name='ID', primary_key=True, auto_created=True)
```

Reply:
I've changed the quoting, but isn't this a good place to use f-strings? To me it's certainly more readable.